clicommand/agent_start.go

```diff
@@ -724,6 +726,16 @@ var AgentStartCommand = cli.Command{
 			Usage:  "Enable debug logging for pipeline signing. This can potentially leak secrets to the logs as it prints each step in full before signing. Requires debug logging to be enabled",
 			EnvVar: "BUILDKITE_AGENT_DEBUG_SIGNING",
 		},
+		cli.StringSliceFlag{
+			Name:   "signing-ignored-env-vars",
+			Usage:  "A list of environment variable names to ignore when signing the pipeline. These variables (and their values) will not be included in the signature. For signatures to match, the corresponding flag must be set on the agent that verifies the signature and runs the job",
+			EnvVar: "BUILDKITE_AGENT_SIGNING_IGNORED_ENV_VARS",
+		},
+		cli.StringSliceFlag{
+			Name:   "verification-ignored-env-vars",
+			Usage:  "A list of environment variable names to ignore when signing the pipeline. These variables (and their values) will not be included in the signature. For signatures to match, the corresponding flag must be set on the agent that verifies the signature and runs the job",
```

Comment:
This is the same usage string as the other new flag? Should it match the comment in agent_configuration.go?